sdks/builds/bcl.mk

```diff
@@ -43,11 +43,15 @@ build-$(1)-bcl: build-bcl
 build-custom-$(1)-bcl:
 	$$(MAKE) -C bcl -C runtime all-mcs build_profiles="$(2)"
 	$$(if $(3),$$(MAKE) -C bcl -C runtime test xunit-test test_profiles="$(3)")
+	$$(MAKE) -C bcl -C mcs/tools/mono-api-html
+	$$(MAKE) -C bcl -C mcs/tools/corcompare
 
 .PHONY: package-$(1)-bcl
 package-$(1)-bcl:
 	$$(foreach profile,$(2), \
 		cp -R $$(TOP)/mcs/class/lib/$$(profile)/* $$(TOP)/sdks/out/$(1)-bcl/$$(profile);)
+	cp $$(TOP)/mcs/tools/mono-api-html/mono-api-html.exe $$(TOP)/sdks/out/$(1)/
+	cp $$(TOP)/mcs/tools/corcompare/mono-api-info.exe $$(TOP)/sdks/out/$(1)/
```

Comment:
We should add these tools to the `monodroid_tools` profile instead of building/copying them here.

Reply:
Oh that's even better, I forgot about that profile - we briefly discussed a new tools dir.

Reply:
Hmm they are already there: https://github.com/mono/mono/blob/aed21f3e4bcdc2f50640866affe808cc5c1423ed/mcs/tools/Makefile#L56

Reply:
I see, they are on master, not 2019-02. I'll double-check in the archive but maybe we're good here then :)

Reply:
Yep, as of master they are in `android-bcl/monodroid/monodroid_tools`, as is `pdb2mdb.exe`